### DISCUSSION TITLE
How to upload files in a repo?

### DISCUSSION THREAD

**Discussion by SriRam-Macha** [SpinlockLabs/github.dart#258] [Q&A]
var github = GitHub(auth: Authentication.withToken('token'));
Repository repo = await github.repositories.getRepository(RepositorySlug('acc-name', 'repo-name'));
now what cant understand

**Comment by Zebak2021** [comment]
![BingWallpaper (1).jpg](https://user-images.githubusercontent.com/93508527/155537819-145d6c6c-b6df-48a7-a5e0-6432f07cede6.jpg)
